source/code/troubleshooter/omsagent_tsg_zip/tsg_files/tsg_code/syslog1/tsg_checkconf.py

```diff
@@ -0,0 +1,161 @@
+import os
+import re
+
+from tsg_errors           import tsg_error_info
+from tsg_info             import tsginfo_lookup
+from install.tsg_checkoms import comp_versions_ge, get_oms_version
+
+syslogconf_path = "/etc/opt/microsoft/omsagent/sysconf/omsagent.d/syslog.conf"
```

Comment:
why is this in `/sysconf/` and not the `workspaceid/conf` directory?

Reply:
Mistake on my part, updated the correct path:
https://github.com/microsoft/OMS-Agent-for-Linux/blob/bff7c80f869d6f5a4046ed41b4bd0ed89d69219f/source/code/troubleshooter/modules/syslog1/check_conf.py#L10